Python/gc_free_threading.c

```diff
@@ -609,6 +640,25 @@ clear_weakrefs(struct collection_state *state)
 {
     PyObject *op;
     WORKSTACK_FOR_EACH(&state->unreachable, op) {
+        if (PyGen_CheckExact(op) ||
+            PyCoro_CheckExact(op) ||
+            PyAsyncGen_CheckExact(op)) {
```

Comment:
Maybe format it something like the following with `{` on it's own line:

```suggestion
        if (PyGen_CheckExact(op) || PyCoro_CheckExact(op) ||
            PyAsyncGen_CheckExact(op))
        {
```